PolicyServicePkg/README.md

```diff
@@ -207,19 +207,19 @@ Like the PEIM, the DXE driver will install/reinstall a NULL protocol with the
 given policies GUID when it is created or updated to allow for notification and
 dispatch on the policy availability.
 
-### Standalone MM
+### Traditional/Standalone MM

```

Comment:
Can this commit just be part of the original PolicyService commit? In case this doesn't come through well, the update to support SMM.